compilers/concrete-compiler/compiler/lib/Support/Pipeline.cpp

```diff
@@ -345,15 +346,33 @@ normalizeTFHEKeys(mlir::MLIRContext &context, mlir::ModuleOp &module,
 mlir::LogicalResult
 extractTFHEStatistics(mlir::MLIRContext &context, mlir::ModuleOp &module,
                       std::function<bool(mlir::Pass *)> enablePass,
-                      CompilationFeedback &feedback) {
-  mlir::PassManager pm(&context);
-  pipelinePrinting("TFHEStatistics", pm, context);
+                      ProgramCompilationFeedback &feedback) {
+  auto funcs = module.getOps<mlir::func::FuncOp>();
```

Comment:
what is the motivation of not doing that in the pass and keeping the pass as module pass?

Reply:
Changed it